wordpress-mobile/gutenberg-mobile
# Update more block styling

Fixes https://github.com/wordpress-mobile/gutenberg-mobile/issues/318 . This PR includes changes for updating styling of more block as below.

<img width="342" alt="screen shot 2018-12-10 at 20 51 31" src="https://user-images.githubusercontent.com/5032900/49751897-b9be2700-fcbf-11e8-8e05-6101c7eb9054.png">

Code changes are in the [child PR](https://github.com/WordPress/gutenberg/pull/12767).

**To Test**

- Open the example app and add a More block
- Verify "Read more" text has continuous lines around
- Verify that you can modify "Read More" text and you can enter long text with multiple lines
